src/builders/mod.rs

```diff
@@ -4,7 +4,11 @@
 mod class;
 mod function;
 mod module;
+#[cfg(feature = "embed")]
+mod sapi;
 
 pub use class::ClassBuilder;
 pub use function::FunctionBuilder;
 pub use module::ModuleBuilder;
+#[cfg(feature = "embed")]
+pub use sapi::SapiBuilder;
```

Comment:
could it be its own feature decoupled from the `embed` one?

Reply:
It should be possible be it's useless since you use SAPI when you build a binary, and in this case you need to be linked to libphp which is only available with the embed feature